### DISCUSSION TITLE
Enable Control Plane Logging via EKS Blueprint

### DISCUSSION THREAD

**Discussion by jwilms1971** [aws/aws-cdk#27713] [Q&A]
I am trying to enable control plane logging using CDK with EKS Blueprints.

const blueprint = blueprints.EksBlueprint.builder()
    .account(account)
    .region(region)
    .addOns(...addOns)
    .clusterProvider(clusterProvider)
    .enableControlPlaneLogTypes([{ControlPlaneLogType: 'api'},{ControlPlaneLogType: 'audit'}])
    .useDefaultSecretEncryption(true) // set to false to turn secret encryption off (non-production/demo cases)
    .build(app, clusterName);

I think I am getting the syntax for enableControlPlaneLogTypes method wrong (I did import ControlPlaneLogType), and was hoping for some guidance. I am still very new to CDK.

**Comment by arminhs** [comment]
This is the correct syntax:

```ts
.enableControlPlaneLogTypes(ControlPlaneLogType.API, ControlPlaneLogType.AUDIT)
```
